Libplanet.Explorer/Controllers/ExplorerController.cs

```diff
@@ -37,8 +39,9 @@ public IActionResult GetRoot()
         {
             var json = _schema.Execute(_ =>
             {
-                _.UserContext = BlockChain;
+                _.UserContext = (BlockChain, Store);
```

Comment:
Rather than an ad-hoc tuple, how about defining a new type having two fields, say, `ExplorerContext` instead?

Reply:
Okay. To achieve it, we can use `IBlockChainContext<T>` type directly instead of defining a new type.